inference-engine/src/transformations/src/transformations/serialize.cpp

```diff
@@ -65,7 +67,12 @@ class XmlVisitor : public ngraph::AttributeVisitor {
     }
     void on_adapter(const std::string& name,
                     ngraph::ValueAccessor<std::string>& adapter) override {
-        m_data.append_attribute(name.c_str()).set_value(adapter.get().c_str());
+        if (name == "__generic_ie_type__") {
```

Comment:
Can we add comment that this construction allow us to avoid serialization of `__generic_ie_type__` attribute?

Reply:
Done in this [commit](https://github.com/openvinotoolkit/openvino/pull/2862/commits/3b89c8d31af7452a364f7d1695ecaa2598e4d63e)